extensions/ql-vscode/package.json

```diff
@@ -778,6 +784,11 @@
           "when": "view == codeQLDatabasesExperimental",
           "group": "navigation"
         },
+        {
+          "command": "codeQLDatabasesExperimental.addNewDatabase",
+          "when": "view == codeQLDatabasesExperimental",
```

Comment:
Once you've rebased/merged to include the changes from https://github.com/github/vscode-codeql/pull/1866, you can add an [extra condition](https://github.com/github/vscode-codeql/pull/1866/files#diff-be4c5a31505df7de81fa4900e84e66c04d1a6bca9b3da50f8061d2cb2bfbbde1R783) which will hide the button if the config is undefined! 😊